test/query.spec.js

```diff
@@ -195,42 +195,50 @@ describe('query.execute()', () => {
   });
 
   describe('update', () => {
-    it('should insert some data', () =>
-      execute(`insert data {:foo :bar :baz}`)
+    let clearDatabase = () => execute('clear all');
```

Comment:
Actually you don't even need to make this a variable, just do `afterEach(() => execue('clear all'))`